fabric-level-generators-v1/src/main/java/net/fabricmc/fabric/impl/level/generator/FabricLevelGeneratorType.java

```diff
@@ -33,8 +34,8 @@
 	public static final HashMap<LevelGeneratorType, ChunkGeneratorSupplier> suppliers = new HashMap<>();
 
 	public static LevelGeneratorType create(Identifier name, Identifier storedName, int version, ChunkGeneratorType<?, ? extends ChunkGenerator<?>> generatorType, Function<World, BiomeSource> biomeSource) {
-		if (changeIdentifierSeparator(name).contains(":") || name.getNamespace().contains(".")) {
-			throw new IllegalArgumentException("Character ':' is not allowed in level generator type identifier and '.' is not allowed in namespace");
+		if (name.getNamespace().contains(".")) {
+			throw new IllegalArgumentException("Character '.' is not allowed in namespace");
 		}
```

Comment:
Doesn't the `Identifier` constructor already prevent this?

Reply:
Yes, it does.

Reply:
No?
Identifier.isNamespaceValid
46 is '.'

Reply:
oh huh, apparently it's just `/` that's disallowed in namespace. Why are we forbidding `.` in namespace, then?

Reply:
https://github.com/FabricMC/fabric/pull/483/files#diff-23dfd92592cd81a56560256a64a9bf76R79
So we can know which part is namespace.

Reply:
are you not able to use a colon there?

Reply:
It was colon before, but to keep everything same with lang it became dot.

Reply:
...but you could just have an overridden `getTranslationKey` that converts it to a period.

Reply:
It was like that before, I liked that more.

Reply:
it can go back to that, can't it?

Reply:
Should be good now.